signac/contrib/job.py

```diff
@@ -359,11 +355,36 @@ def _statepoint_filename(self):
         # use str.join with os.sep instead of os.path.join for speed.
         return os.sep.join((self.workspace(), self.FN_MANIFEST))
 
-    @property
+    # Decorated properties aren't supported: https://github.com/python/mypy/issues/1362
+    @property  # type: ignore
```

Comment:
```suggestion
    # Adding '# type: ignore' for mypy, because decorated properties aren't supported:
    # https://github.com/python/mypy/issues/1362
    @property  # type: ignore
```